.github/workflows/build.yml

```diff
@@ -87,21 +87,21 @@ jobs:
           TAGS="--tag ${DOCKER_IMAGE}:${VERSION}"
           TAGS_RHEL="--tag ${DOCKER_IMAGE}:${VERSION}-rhel"
 
-          echo ::set-output name=DOCKERHUB_USERNAME::${DOCKERHUB_USERNAME}
-          echo ::set-output name=docker_repo::${DOCKER_REPO}
-          echo ::set-output name=docker_image::${DOCKER_IMAGE}
-          echo ::set-output name=version::${VERSION}
-          echo ::set-output name=buildx_args::--platform ${DOCKER_PLATFORMS} \
+          echo "DOCKERHUB_USERNAME=${DOCKERHUB_USERNAME}"  >> $GITHUB_OUTPUT
+          echo "docker_repo=${DOCKER_REPO}"  >> $GITHUB_OUTPUT
+          echo "docker_image=${DOCKER_IMAGE}" >> $GITHUB_OUTPUT
+          echo "version=${VERSION}"  >> $GITHUB_OUTPUT
+          echo "buildx_args=--platform ${DOCKER_PLATFORMS} \
             --build-arg VERSION=${VERSION} \
             --build-arg BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ') \
             --build-arg VCS_REF=${GITHUB_SHA::8} \
-            ${TAGS} --file build/dockerfiles/Dockerfile .
+            ${TAGS} --file build/dockerfiles/Dockerfile .  >> $GITHUB_OUTPUT
 
-          echo ::set-output name=buildx_args_rhel::--platform linux/amd64 \
+          echo "buildx_args_rhel=--platform linux/amd64 \
             --build-arg VERSION=${VERSION} \
             --build-arg BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ') \
             --build-arg VCS_REF=${GITHUB_SHA::8} \
-            ${TAGS_RHEL} --file build/dockerfiles/rhel.Dockerfile .
+            ${TAGS_RHEL} --file build/dockerfiles/rhel.Dockerfile .  >> $GITHUB_OUTPUT
       -
         name: Set up Docker Buildx
         uses: crazy-max/ghaction-docker-buildx@v3.2.0
```

Comment:
on line 113, why do we build the image (but not push it) then on line 122, we build it AGAIN and this time push it?

and is `DOCKERHUB_PASSWORD` really for dockerhub, or are we actually pushing to quay here? 

I realize these are out of scope of the specific issue at hand, but it would be valuable to also fix these in a second PR (or a second commit on this PR), since they're easy to solve at the same time.

Reply:
looks like there will be a more global rework of configbump workflows with this PR https://github.com/che-incubator/configbump/pull/76/files